2 versions of tensorboard get installed with SCT

When SCT is installed, 2 versions of tensorboard get installed:

* ~~`tensorboard==2.3.0`~~ `tensorboard==2.4.0` (i.e. the latest version), which gets installed [because of `ivadomed`](https://github.com/ivadomed/ivadomed/blob/master/requirements.txt#L13).
* `tensorflow-tensorboard==1.5.1`, which is ["a relic of a bygone era; there’s no need to install it anymore."](https://github.com/tensorflow/tensorboard/issues/1862#issuecomment-521876133)
    * The reason the older, differently-named `tensorflow-tensorboard` gets installed is because it's a dependency for `tensorflow==1.5.0`. (The newer-named `tensorboard` was only [introduced in Tensorflow 1.6.0+](https://pypi.org/project/tensorflow-tensorboard/).)

**This has the potential to create conflicts.** For example, it is currently roadblocking #3034. An unrelated import from `ivadomed` triggers a new exception [in this CI build](https://travis-ci.org/github/neuropoly/spinalcordtoolbox/jobs/743004132#L2003):

```
unit_testing/test_deepseg.py:16: in <module>
    import sct_deepseg
scripts/sct_deepseg.py:18: in <module>
    from ivadomed import inference as imed_inference
python/envs/venv_sct/lib/python3.6/site-packages/ivadomed/inference.py:16: in <module>
    from ivadomed import training as imed_training
python/envs/venv_sct/lib/python3.6/site-packages/ivadomed/training.py:13: in <module>
    from torch.utils.tensorboard import SummaryWriter
python/envs/venv_sct/lib/python3.6/site-packages/torch/utils/tensorboard/__init__.py:2: in <module>
    from tensorboard.summary.writer.record_writer import RecordWriter  # noqa F401
python/envs/venv_sct/lib/python3.6/site-packages/tensorboard/summary/__init__.py:31: in <module>
    from tensorboard.summary import v2
python/envs/venv_sct/lib/python3.6/site-packages/tensorboard/summary/v2.py:24: in <module>
    from tensorboard.plugins.audio.summary_v2 import audio
python/envs/venv_sct/lib/python3.6/site-packages/tensorboard/plugins/audio/summary_v2.py:30: in <module>
    from tensorboard.compat import tf2 as tf
python/envs/venv_sct/lib/python3.6/site-packages/tensorboard/compat/__init__.py:28: in <module>
    import tensorboard.lazy as _lazy
E   AttributeError: module 'tensorboard' has no attribute 'lazy'
```

This exception is specifically [because 2 versions are installed](https://github.com/tensorflow/tensorboard/issues/1862#issuecomment-521874990). Ideally, we should use just `tensorboard`, but our `tensorflow==1.5.0` requirement is preventing that.

--- a/install_sct
+++ b/install_sct
@@ -533,16 +533,18 @@ print info "Installing Python dependencies..."
 # Check if a frozen version of the requirements exist (for release only)
 if [[ -f "requirements-freeze.txt" ]]; then
   print info "Using requirements-freeze.txt (release installation)"
-  pip install -r requirements-freeze.txt
+  export REQUIREMENTS_FILE="requirements-freeze.txt"
 else
   # Not a package
   print info "Using requirements.txt (git installation)"
-  pip install -r requirements.txt
-  yes | pip uninstall tensorflow-tensorboard
-  yes | pip uninstall tensorboard
-  pip install tensorboard
-
+  export REQUIREMENTS_FILE="requirements.txt"
 fi
+pip install -r REQUIREMENTS_FILE &&
+  # `tensorflow-tensorboard` is installed by `tensorflow==1.5.0` but is not needed,
+  # and conflicts with the other installation of `tensorboard`. So, we uninstall here.
+  # This is hacky, and should be removed as soon as we stop using `tensorflow==1.5.0`.
+  # See https://github.com/neuropoly/spinalcordtoolbox/issues/3035 for more info.
+  yes | pip uninstall tensorflow-tensorboard
 if [[ $? != 0 ]]; then
   die "Failed running pip install: $?"
 fi

--- a/install_sct
+++ b/install_sct
@@ -539,7 +539,7 @@ else
   print info "Using requirements.txt (git installation)"
   export REQUIREMENTS_FILE="requirements.txt"
 fi
-pip install -r REQUIREMENTS_FILE &&
+pip install -r $REQUIREMENTS_FILE &&
   # `tensorflow-tensorboard` is installed by `tensorflow==1.5.0` but is not needed,
   # and conflicts with the other installation of `tensorboard`. So, we uninstall here.
   # This is hacky, and should be removed as soon as we stop using `tensorflow==1.5.0`.

--- a/install_sct
+++ b/install_sct
@@ -544,7 +544,9 @@ pip install -r $REQUIREMENTS_FILE &&
   # and conflicts with the other installation of `tensorboard`. So, we uninstall here.
   # This is hacky, and should be removed as soon as we stop using `tensorflow==1.5.0`.
   # See https://github.com/neuropoly/spinalcordtoolbox/issues/3035 for more info.
-  yes | pip uninstall tensorflow-tensorboard
+  yes | pip uninstall tensorflow-tensorboard &&
+  yes | pip uninstall tensorboard &&
+  pip install tensorboard
 if [[ $? != 0 ]]; then
   die "Failed running pip install: $?"
 fi

